packages/matchbox-icons/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@sparkpost/matchbox-icons",
+  "version": "1.0.0",
+  "description": "React icons for Matchbox",
+  "main": "matchbox-icons.js",
+  "module": "matchbox-icons.es.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/SparkPost/matchbox.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "SparkPost",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/SparkPost/matchbox/issues"
+  },
+  "homepage": "https://github.com/SparkPost/matchbox/tree/master/packages/matchbox-icons#readme",
+  "sideEffects": false,
+  "scripts": {
+    "build": "babel-node scripts/build.js --presets=node6"
+  },
+  "dependencies": {
+    "react": "^16.3.2",
+    "react-dom": "^16.3.2"
```

Comment:
Should relax these dependencies to the actual versions needed to use the icons.

Reply:
other component libs seem to use `^15.3.0 || ^16.0.0`, will go with that